pod/perlhacktips.pod

```diff
@@ -1710,6 +1712,10 @@ no extra SV allocations are introduced by enabling the logging.  If
 compiled with C<-DDEBUG_LEAKING_SCALARS>, the serial number for each SV
 allocation is also logged.
 
+The C<c> option uses the C<Perl_c_backtrace> facility, and therefore
+additionally requires the C<-DUSE_C_BACKTRACE> compile flag in order to
```

Comment:
This should probably refer to the Configure -Dusecbacktrace flag instead, which will check for the required library support.